frontend/src/entry/overview/EntryCardEntityRoleRow.tsx

```diff
@@ -0,0 +1,45 @@
+import GroupIcon from '@mui/icons-material/Groups'
+import UserIcon from '@mui/icons-material/Person'
+import { Grid, Stack } from '@mui/material'
+import { Fragment, useMemo } from 'react'
+import UserDisplay from 'src/common/UserDisplay'
+import EntryCardRolesChipSet from 'src/entry/overview/EntryCardRolesChipSet'
+import { CollaboratorEntry } from 'types/types'
+
+type EntryCardEntityRoleRowProps = {
+  entryCollaborators: CollaboratorEntry[]
+}
+
+export default function EntryCardEntityRoleRow({ entryCollaborators }: EntryCardEntityRoleRowProps) {
+  return entryCollaborators.map((user) => (
+    <Fragment key={user.entity}>
+      <Grid item xs={6}>
+        <Stack direction='row' alignItems='center' spacing={1}>
+          <EntryCollaboratorIcon entity={user} />
+          <EntryCollaboratorName entity={user} />
+        </Stack>
+      </Grid>
+      <Grid item xs={6}>
+        <EntryCardRolesChipSet entryCollaborator={user} />
+      </Grid>
+    </Fragment>
+  ))
+}
+
+type EntryCollaboratorProps = {
+  entity: CollaboratorEntry
+}
+
+function EntryCollaboratorIcon({ entity }: EntryCollaboratorProps) {
+  const isUser = useMemo(() => entity.entity.startsWith('user:'), [entity])
+  return isUser ? <UserIcon color='primary' /> : <GroupIcon color='secondary' />
+}
+
+function EntryCollaboratorName({ entity }: EntryCollaboratorProps) {
+  const entityName = useMemo(() => entity.entity.split('/').pop()!, [entity])
+  return (
+    <>
```

Comment:
Again this will be solved by reusing `EntityNameDisplay`, but this fragment isn't needed.

The reason why we use fragments is because in react you must always return only one root element from any function. If we find ourselves in a situation where we have multiple root elements then that's where we need to use a fragment. You could also use something like a `div`, but fragments have the benefit of being removed by react before the DOM is updated, so they're always preferred in these scenarios.

Here's an example of the above:
```
// Invalid
return (
  <div>Hello</div>
  <div>World</div>
)

// Valid
return (
  <>
    <div>Hello</div>
    <div>World</div>
  </>
)

// Also valid
return (
  <Fragment>
    <div>Hello</div>
    <div>World</div>
  </Fragment>
)
```